### PR TITLE
Prac 08 feedback

### PR DESCRIPTION
**Pull request by a-rosa**
@nora-zyy
@AdriantoHartono
@JevanChandra
@Zunidzk

### CODE REVIEW

File: prac_08/box_layout.kv
Comment (AdriantoHartono) [inline prac_08/box_layout.kv:1]: Great code 👌

File: prac_08/box_layout_demo.py
Comment (AdriantoHartono) [inline prac_08/box_layout_demo.py:1]: Nice code👍

File: prac_08/convert_m_km.kv
Comment (AdriantoHartono) [inline prac_08/convert_m_km.kv:1]: Great!

File: prac_08/convert_m_km.py
Comment (AdriantoHartono) [inline prac_08/convert_m_km.py:1]: Nice code, and has been written neatly 👍

File: prac_08/dynamic_labels.kv
Comment (AdriantoHartono) [inline prac_08/dynamic_labels.kv:1]: 👌

File: prac_08/dynamic_labels.py
Comment (AdriantoHartono) [inline prac_08/dynamic_labels.py:1]: Great code🙏

File: prac_08/squaring.kv
Comment (AdriantoHartono) [inline prac_08/squaring.kv:1]: Nice code

File: prac_08/squaring.py
Comment (AdriantoHartono) [inline prac_08/squaring.py:1]: Great👍

File: prac_08/convert_m_km.py
Comment (nora-zyy) [reply prac_08/convert_m_km.py:1]: It's easy and clear.

File: prac_08/squaring.kv
Comment (nora-zyy) [reply prac_08/squaring.kv:1]: Very concise completion of the questions required.

**Pull request merged**